lib/ReactViews/Map/Panels/SharePanel/SharePanel.tsx

```diff
@@ -80,9 +79,8 @@ class SharePanel extends React.Component<PropTypes, SharePanelState> {
             terria={terria}
             viewState={viewState}
             includeStories={true}
-            shouldShorten={
-              !!terria.getLocalProperty("shortenShareUrls") ?? true
-            }
+            // Note we don't use terria.getLocalProperty("shortenShareUrls") here - as it is only configurable in SharePanelContent
+            shouldShorten={!!canShorten(terria)}
```

Comment:
Maybe check if local property is undefined and then use `canShorten` otherwise use local property value, so user choice doesn't get overridden?

Reply:
Done!